src/main/java/picard/illumina/IlluminaBasecallsToFastq.java

```diff
@@ -176,6 +176,9 @@
             "is larger than the available memory.")
     public Boolean FORCE_GC = true;
 
+    @Argument(doc = "If true, the output records are sorted by read name. Otherwise they are unsorted.")
```

Comment:
I think we should add, either here (and in IBTS) or in the general usage that setting `SORT=false` doesn't produce unsorted data, but that it produces data in the order that the seuqencer (and bcl2fastq2) produce it - i.e. ordered by tile and within tile by position, but not with a full queryname sort.